--- /usr/share/debootstrap/functions	2023-06-24 07:12:13.000000000 +0000
+++ functions	2023-07-21 12:28:20.171303531 +0000
@@ -627,7 +627,11 @@
 
 	progress 0 100 DOWNREL "Downloading Release file"
 	progress_next 100
-	if [ -n "$INRELEASE_PATH" ]; then
+        if [ -n "$LOCAL_INRELEASE_PATH" ]; then
+		cp "$LOCAL_INRELEASE_PATH" "$inreldest"
+                split_inline_sig "$inreldest" "$reldest" "$relsigdest"
+		progress 100 100 DOWNREL "Got InRelease file locally"
+	elif [ -n "$INRELEASE_PATH" ]; then
 		get "$m1/dists/$suite/$INRELEASE_PATH" "$inreldest" nocache ||
 			error 1 NOGETREL "Failed getting release file %s" \
 			"$m1/dists/$suite/$INRELEASE_PATH"
